src/plugin/file/kml/ui/kmlui.js

```diff
@@ -110,6 +110,9 @@ plugin.file.kml.ui.createOrEditPlace = function(options) {
   };
 
   var label = (options['feature'] ? 'Edit' : 'Add') + ' Place';
+  if (os.ui.window.exists(windowId)) {
+    os.ui.window.bringToFront(windowId);
+  } else {
   var windowOptions = {
```

Comment:
This section isn't tabbed in, causing lint errors. Do you have eslint configured in your editor?